src/stacktrace_windows.cpp

```diff
@@ -23,6 +23,8 @@
 #include <mutex>
 #include <g3log/g3log.hpp>
 
+#pragma comment(lib, "dbghelp.lib")
```

Comment:
Can we just Link with dbghelp.lib?

Ref: https://github.com/KjellKod/g3sinks/blob/master/logrotate/test/Test.cmake

I don't have access to Windows so I can't test either. But I figured if it is enough to just link it then for Windows users it would a done deal?